packages/terra-functional-testing/src/config/wdio.conf.js

```diff
@@ -135,6 +135,13 @@ exports.config = {
     ui: 'bdd',
     timeout: 1200000,
   },
+  /**
+   * Defines the defaullt repository id and url used for uploading and downloading reference screenshots from a remote repository.
+   */
+  screenshotsSites: {
+    repositoryId: 'cerner-release-main-site',
```

Comment:
I don't think we should hard-code Cerner-specific values in our OSS repos. No one outside of the company will be able to use them. How do we typically handle cases like this where we know we're the primary audience and consumer yet the code is in our open source repos?

One idea is to leave these two values (also line 143) undefined here, then later provide those values either in an internal wrapper project. A second option is to provide these values as environmental variables at runtime.

Lastly, if we don't mind putting values like this in our OSS repos then I guess it's fine to keep these as-is. It just seems odd to mix the two worlds. Kinda smells like we are not correctly separating the concerns between our projects.

Reply:
@xenoworf This makes perfect sense. I think we should not provide a default and always require the consumer to provide the repository id and url if using this feature.
https://github.com/cerner/terra-toolkit/pull/740/commits/4b6b71dd14d9a363ef49d27716c7b3cd1fb3c7b4